README.md

```diff
@@ -148,6 +148,19 @@ command \
 ```
 This takes inspiration from the Eclipse feature.
 
+## How to install the sublime-text plugin
```

Comment:
Can we just list this on that Sublime package manager thing? I forget the name, I used it a long time ago.

Reply:
@lovesegfault I am aware of that (https://github.com/wbond/package_control_channel/). But for this, the code of the plugin must be present in this repo beforehand. Then, I just need to provide this information (url of this repo, name of the plugin etc) to the package control channel. I will take on that in the next phase after this commit is in.

Reply:
Awesome, in this case I prefer we leave these instructions out until we can just tell the user to use PackageControl :)

Reply:
Sure!